cloud/shared/bin/lib/config_loader.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+
+"""
+Config Loader
+  Handles validating and getting data from the configuration/variable files
+  Inputs:
+    - config_filename: a json object with keys being the 
+    variable names and values being the values to set
+    - variable_definition_filename: a json object with keys being the 
+    variable names and values being an metadata object about that 
+    variable (see the variable_definitions.json in the /shared)
+  Call load_files to 
+"""
+
+
+class ConfigLoader:
+    def __init__(self, config_filename, variable_definition_filename):
+        self.config_filename = config_filename
+        self.variable_definition_filename = variable_definition_filename
```

Comment:
issue (blocking): there will be at least two, but probably more variable definition files for a given deployment: the shared variables and the cloud provider-specific variables. If we factor things well there'll probably also be separate variable definition files for each swappable dependency one for LoginRadius, one for IDCS, one for AWS SES, etc